rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/object/MetaFieldGenerator.xtend

```diff
@@ -201,9 +195,9 @@ class MetaFieldGenerator {
 		buildClass(packages.basicMetafields, body, scope)
 	}
 
-	def CharSequence fieldWithMeta(RootPackage root, JavaClass<?> metaJavaType, RType valueType) {
+	private def CharSequence fieldWithMeta(RootPackage root, RJavaFieldWithMeta metaJavaType, RMetaAnnotatedType metaAnnotatedType) {
```

Comment:
Would be somewhat more beautiful if `RType valueType` did not change.

I.e., pass in the `RType`, and then do `valueType.withEmptyMeta` and `valueType.toJavaReferenceType` below. The code needs less changes than done here.